ramls/transaction-summary.raml

```diff
@@ -0,0 +1,59 @@
+#%RAML 1.0
+
+title: Finance Transaction Summaries
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: https://github.com/folio-org/mod-finance
+
+documentation:
+  - title: Transaction Summaries API
+    content: This documents the API calls that can be made to create Transaction summaries for Orders and Invoices
+
+types:
+  order-transaction-summary: !include acq-models/mod-finance/schemas/order_transaction_summary.json
+  errors: !include raml-util/schemas/errors.schema
+  
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+  language: !include raml-util/traits/language.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/finance/order-transaction-summaries:
+  post:
```

Comment:
If this is only POST then why not use `ramls/rtypes/post-json-201.raml`?

Reply:
changed